.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and verify PR Title
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
I guess `branches` can be removed? and instead, you can specify the types
types:
      - opened
      - edited
So that on title update also it runs the semantic action again. Currently that might not be happening

Reply:
Done